src/NzbDrone.Core/MediaFiles/AudioTagService.cs

```diff
@@ -90,7 +90,7 @@ private AudioTag GetTrackMetadata(TrackFile trackfile)
                 OriginalReleaseDate = album.ReleaseDate,
                 OriginalYear = (uint)album.ReleaseDate?.Year,
                 Publisher = release.Label.FirstOrDefault(),
-                MusicBrainzReleaseCountry = IsoCountries.Find(release.Country.FirstOrDefault()).TwoLetterCode,
+                MusicBrainzReleaseCountry = IsoCountries.Find(release.Country.FirstOrDefault())?.TwoLetterCode,
```

Comment:
Does the find fail if the inner comes back null?

Reply:
No, find has `null` checking built in, and will return `null` if passed in `null`
https://github.com/lidarr/Lidarr/blob/develop/src/NzbDrone.Core/Parser/IsoCountries.cs#L272